draft-kuepau-iab-privacy-partitioning.md

```diff
@@ -454,7 +436,42 @@ about individual client data.
 
 # Applying Privacy Partioning
 
-TODO: talk about how to think about new protocols that are using partitioning, the models that we derive from looking at the existing ones, how you recognize a protocol that's using this, and what tools do we have to evaluate it
+Applying privacy partitioning to an existing or new system or protocol requires the following steps:
+
+1. Identify the types of information used or exposed in a system or protocol, some of which can be used to identify a user or correlate to other contexts.
+1. Partition different types of user-identifying or correlatable information across contexts to minimize the amount of
+user-identifying or correlatable information in any single context.
+
+The most impactful types of user-identifying information to partition are (a) user identity or identities that can be linked and (b) user data.
+For example, Oblivious DoH and Oblivious HTTP partition the client IP address and client request data into
+separate contexts, thereby ensuring that no entity beyond the client can observe both. Collusing across contexts
+may reverses this partition process, but can also promote non-user-identifying information to user-identifying.
+For example, in CONNECT proxy systems that use QUIC, the QUIC connection ID is inherently non-user-identifying
+since it is generated randomly {{QUIC, Section 5.1}}. However, if combined with another context that has user-identifying
+information such as the client IP address, the QUIC connection ID can become user-identifying information.
+
+This partitioning process can be applied incorrectly or incompletely. Partitions may contain
+more user-identifying information than desired, or some information in a context may be more user-identifying
+than intended. Nevertheless, partitions can help improve the client's privacy posture when applied carefully.
+
+Evaluating and qualifying the resulting privacy of a system or protocol that applies privacy partitioning depends
+on the contexts that exist and types of user-identifying information in each context. Such evaluation is
```

Comment:
I proposing this change because it is not at all clear what is means to evaluate the context that exists. I would say this evaluation is actually to identify the contexts itself, which means you identify all entities that below to a context and the information that is shared in this context.